src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs

```diff
@@ -102,9 +102,9 @@ ResourceManifest BuildManifest()
             manifest
                 .DefineScript("jQuery-ui-i18n")
                 .SetDependencies("jQuery-ui")
-                .SetUrl("~/OrchardCore.Resources/Scripts/jquery-ui-i18n.min.js", "~/OrchardCore.Resources/Scripts/jquery-ui-i18n.js")
-                .SetCdn("https://code.jquery.com/ui/1.7.2/i18n/jquery-ui-i18n.min.js", "https://code.jquery.com/ui/1.7.2/i18n/jquery-ui-i18n.min.js")
```

Comment:
@deanmarcussen I remove it because the CDN has the minified version only, so I'm not sure how the source **~/OrchardCore.Resources/Scripts/jquery-ui-i18n.js** has been added?!!

Seems it added accidently by the line above

Reply:
Please remove the change

Reply:
No problem I will remove it but can anyone let me know from where this file is coming while the CDN points to the **https://code.jquery.com/ui/1.7.2/i18n/jquery-ui-i18n.min.js**?